test/dest-symlinks.js

```diff
@@ -33,11 +34,14 @@ describe('.dest() with symlinks', function() {
   beforeEach(clean);
   afterEach(clean);
 
-  it('creates symlinks when the `symlink` attribute is set on the file', function(done) {
+  it('creates symlinks when `file.isSymbolic()` is true', function(done) {
     var file = new File({
       base: inputBase,
       path: inputPath,
       contents: null,
+      stat: {
+        isSymbolic: isSymbolic,
```

Comment:
This seems wrong -- we'll want to mock either `file.isSymbolic()` or `file.stat.isSymbolicLink()` right?

https://nodejs.org/api/fs.html#fs_class_fs_stats
https://github.com/gulpjs/vinyl/blob/master/index.js#L84